tests/unit_tests/docmaps/v2/codecs/elife_manuscript_test.py

```diff
@@ -121,22 +124,23 @@ def test_should_not_populate_published_if_rp_publication_timestamp_is_none(self)
 class TestGetDocmapElifeManuscriptOutputForVor:
     def test_should_populate_docmaps_elife_manuscript_output_for_vor(self):
         result = get_docmap_elife_manuscript_output_for_vor(
-            query_result_item=DOCMAPS_QUERY_RESULT_ITEM_1,
-            manuscript_version=MANUSCRIPT_VERSION_1
+            query_result_item=DOCMAPS_QUERY_RESULT_ITEM_WITH_VOR_VERSION,
+            manuscript_version=MANUSCRIPT_VOR_VERSION_1
         )
         manuscript_version_doi = get_elife_manuscript_version_doi(
-            elife_doi=DOCMAPS_QUERY_RESULT_ITEM_1['elife_doi'],
-            elife_doi_version_str=MANUSCRIPT_VERSION_1['elife_doi_version_str']
+            elife_doi=DOCMAPS_QUERY_RESULT_ITEM_WITH_VOR_VERSION['elife_doi'],
+            elife_doi_version_str=MANUSCRIPT_VOR_VERSION_1['elife_doi_version_str']
         )
         assert result == {
             'type': 'version-of-record',
             'doi': manuscript_version_doi,
+            'published': VOR_PUBLICATION_DATE_1.strftime('%Y-%m-%d'),
```

Comment:
minor: maybe use `.isoformat()` as that is what we seem to be using in other examples?